src/main/java/DenisCamacho/task_02/Table.java

```diff
@@ -0,0 +1,94 @@
+package main.java.DenisCamacho.task_02;
+
+public class Table {
+    private String[][] table;
+
+    public Table() {
+        table = new String[3][3];
+        cleaning();
+    }
+
+    public void insert(Player player, int row, int column) {
+        table[row][column] = String.format("%s", player.getChip());
+    }
+
+    public void cleaning() {
+        System.out.println("Cleaning the table #################################################");
+        for (int row = 0; row < table.length; row++) {
+            for (int column = 0; column < table[row].length; column++) {
+                table[row][column] = "___";
+            }
+        }
+    }
+
+    public void printer() {
```

Comment:
@denis1729 this method printer should be separated from the table, this is related to your console app, the model only needs to set the values and know if there are some winner